requirements.txt

```diff
@@ -6,6 +6,6 @@ gevent
 requests
 rdflib
 rdflib-jsonld
-mf2py
+mf2py==1.1.0
```

Comment:
I think it is good to run tests on latest mf2py version, not only on 1.1.0. Maybe change it to >=?

Reply:
Actually, `1.1.0` is the latest version of `mf2py`

Reply:
Yes, but if there will be a new version, our tests won't be able to check how it works, we'd have to track these versions manually, updating it from time to time.

Reply:
Sorry, I didn't get your first comment - now I understand :) Sure, will do that :+1: